CMakeLists.txt

```diff
@@ -0,0 +1,219 @@
+cmake_minimum_required(VERSION 3.0.2)
+set(CMP0048 NEW)
+
+project(_tango LANGUAGES CXX
+        VERSION 1.0)
+        
+#need to define the version of the library
+set(MAJOR_VERSION "9")
+set(MINOR_VERSION "2")
+set(PATCH_VERSION "2")
```

Comment:
The patch version needs to be updated :)

Reply:
There are ways to tell CMake to get the project version from git tags, if you are interested, I can provide a  PR.